types/react-mentions/lib/utils/mapPlainTextIndex.d.ts

```diff
@@ -1,17 +1,16 @@
-import { DisplayTransformFunc } from "../";
-
 /**
- * For the passed character index in the plain text string, returns the corresponding index in the marked up value string.
- * If the passed character index lies inside a mention, the value of `inMarkupCorrection` defines the correction to apply:
+ * For the passed character index in the plain text string, returns the
+ * corresponding index in the marked up value string. If the passed character
+ * index lies inside a mention, the value of `inMarkupCorrection` defines the
+ * correction to apply:
  * - 'START' to return the index of the mention markup's first char (default)
  * - 'END' to return the index after its last char
  * - 'NULL' to return null
  */
+
 export function mapPlainTextIndex(
     value: string,
     markup: string,
     indexInPlainText: number,
-    inMarkupCorrection: string,
-    displayTransform: DisplayTransformFunc,
-    regex: RegExp
+    inMarkupCorrection: 'START' | 'END' | 'NULL',
 ): number;
```

Comment:
I did some checking and this was the signature last [as of react-mentions v2.4.2](https://github.com/signavio/react-mentions/blob/v2.4.2/src/utils.js#L161) and the new signature was introduced in v3.0.0 with signavio/react-mentions#308. So I /think/ technically the types version is correct and this signature was just busted for v3.x version before.

However, since there are so many updates, and react-mentions is at v4.x now, I'm wondering if we shouldn't go ahead and [bump the major version of this types package](https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package) to v4.1 (or the oldest version you're confidence these types are compatible with). I think you can just update [the major.minor version number listed in the first line of the index.d.ts file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3ea27f607b34b9bab74940348292c7fffaf3b0b4/types/react-mentions/index.d.ts#L1) from this:

> // Type definitions for react-mentions 3.3

to this

> // Type definitions for react-mentions 4.1

Let me know your thoughts!

Reply:
And in looking back at the beginning of this PR, since we added `a11ySuggestionsListLabel` I think we should bump this to v4.1.

Reply:
@activescott Thanks for your review. 
1. I just followed the documentation to the function and there are missing parameters. See the function https://github.com/signavio/react-mentions/blob/master/src/utils/mapPlainTextIndex.js
2. Thanks. I change the version